docs/source/openapi.json

```diff
@@ -1084,7 +1084,7 @@
       },
       "ColumnType": {
         "type": "string",
-        "enum": ["float", "int", "class_label", "string_label", "string_text", "bool", "list"]
+        "enum": ["float", "int", "class_label", "string_label", "string_text", "bool", "list", "audio"]
```

Comment:
maybe also add an example in /statistics?

Reply:
it would be easier to add a 100%-sure working example when we have them in prod 🥺

Reply:
[feeb31c](https://github.com/huggingface/datasets-server/pull/2612/commits/feeb31cc3feff830642756a267bafc6e2732404f)
but i don't know, do we have an automated way to format openapi specs?

Reply:
Yes, if you use vscode (and the workspace defined here: https://github.com/huggingface/datasets-server/blob/main/.vscode/monorepo.code-workspace), install the recommended extensions (in particular `stoplight.spectral`). Please tell me if it works!

Reply:
eh i use pycharm :upside_down_face: